.github/workflows/netflify.yml

```diff
@@ -0,0 +1,22 @@
+name: Netlify
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
I guess these are the dev builds?

Reply:
yes, it runs against the default branch

Reply:
That was not what you asked :see_no_evil: 
Where it runs does not matter, since the code is not pushed to Netlify, just a webhook to trigger a build.
I don't know what those builds do, but this is a 1:1 replacement of the current Azure pipeline for it.

Reply:
Yeah, commented that below, but see you removed it already :-) Was just thinking that we should probably make the names more clear:

`secrets.NETLIFY_CAST_DEV_BUILD_HOOK`